### ISSUE TITLE
Use the _oldStyle decorator in twisted.internet

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8281]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)| @hawkowl reported|
|-|-|
|Trac ID|trac#8281|
|Type|enhancement|
|Created|2016-04-09 17:38:50Z|
|Branch|https://github.com/twisted/twisted/tree/oldstyle-internet-8281|

Part of [#8243](https://github.com/twisted/twisted/issues/8243)

<details><summary>Searchable metadata</summary>

```
trac-id__8281 8281
type__enhancement enhancement
reporter__hawkowl hawkowl
priority__normal normal
milestone__New_Style_Classes_Only New-Style Classes Only
branch__oldstyle_internet_8281 oldstyle-internet-8281
branch_author__hawkowl hawkowl
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1460223530905063 1460223530905063
changetime__1476175315305403 1476175315305403
version__None None
owner__hawkowl hawkowl

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_2"></a>|@hawkowl commented|
|-|-|

(In [47197]) Branching to oldstyle-internet-8281.

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_4"></a>|@hawkowl commented|
|-|-|

PR: https://github.com/twisted/twisted/pull/549

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_5"></a>|@hawkowl commented|
|-|-|

See PR, please review.

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_6"></a>|@glyph set owner to @hawkowl|
|-|-|

Review at https://github.com/twisted/twisted/pull/549#pullrequestreview-3595823

Suggestions for stuff to split out were more observations for future minimalism, or things that you can do if you want.  If the conflict resolution is trivial and you get a fully passing build, go for it.

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_7"></a>|@hawkowl set status to `closed`|
|-|-|

In changeset 7c48e82f5270fa063d08c5122fd0295908de76e4
```
#!CommitTicketReference repository="" revision="7c48e82f5270fa063d08c5122fd0295908de76e4"
Merge oldstyle-internet-8281: Remove old-style classes in twisted.internet

Author: hawkowl
Reviewer: glyph
Fixes: #8281
```
